<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Huffman_coding</class>
 <widget class="QMainWindow" name="Huffman_coding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>455</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1170</width>
    <height>455</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Huffman_coding</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QTextEdit" name="top_left_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>550</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Строка для кодирования </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="top_right_box">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>550</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблица</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrow_right">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>21</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Дерево</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="bot_right_box">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>550</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="arrow_down">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>550</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>arrow down</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="bot_left_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>550</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сжатая строка (биты)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrow_left">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>21</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrow_up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>550</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>arrow up</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
